packages/yoastseo/src/languageProcessing/index.js

```diff
@@ -62,48 +26,9 @@ const languageConfigs = {
 
 export {
 	AbstractResearcher,
-	transliterate,
-	replaceDiacritics,
-	createRegexFromArray,
-	imageInText,
-	stripSpaces,
-	baseStemmer,
-	getWords,
-	flattenSortLength,
-	indices,
-	buildFormRule,
-	createRulesFromArrays,
-	createSingleRuleFromArray,
-	matchRegularParticiples,
-	directPrecedenceException,
-	precedenceException,
-	nonDirectPrecedenceException,
-	findMatchingEndingInArray,
-	regexHelpers,
-	exceptionListHelpers,
-	stemHelpers,
-	areWordsInSentence,
-	values,
-	getClauses,
-	getClausesSplitOnStopWords,
-	stripHTMLTags,
-	stripBlockTagsAtStartEnd,
-	countMetaDescriptionLength,
-	sanitizeString,
-	removePunctuation,
-	getLanguage,
-	getSentences,
-	getFieldsToMark,
-	unifyAllSpaces,
-	normalizeHTML,
-	collectMarkingsInSentence,
-	normalizeSingle,
-	parseSynonyms,
-	mergeListItems,
-	findWordFormsInString,
-	markWordsInSentences,
-	helpers,
-	researches,
-	languageHelpers,
```

Comment:
I expose these helpers from `languageProcessing/helpers` as `processingHelpers`. This way this `index.js` file is cleaner